src/components/dashboard/main-content/DashboardPopularCourses.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import { InstantSearch } from 'react-instantsearch-dom';
+
+import { Container } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+import { getConfig } from '@edx/frontend-platform/config';
+import { SearchData } from '@edx/frontend-enterprise-catalog-search';
+
+import { PopularResults } from '../../search/popular-results';
+import { COURSE_TITLE, SEARCH_FACET_FILTERS } from '../../search/constants';
+
+import { SubsidyRequestsContextProvider } from '../../enterprise-subsidy-requests';
+
+import { NUM_POPULAR_COURSES_TO_DISPLAY } from './data/constants';
+
+const DashboardPopularCourses = () => {
+  const { algolia } = useContext(AppContext);
+  const config = getConfig();
+
+  return (
+    <SubsidyRequestsContextProvider>
```

Comment:
I think we don't need `SubsidyRequestsContextProvider` here, `DashboardMainContent` is already wrapped.